tests/functional/aws-node-sdk/test/object/getObjectLegalHold.js

```diff
@@ -13,7 +13,10 @@ const unlockedBucket = 'mock-bucket-no-lock';
 const key = 'mock-object-legalhold';
 const keyNoHold = 'mock-object-no-legalhold';
 
-describe('GET object legal hold', () => {
+const isCEPH = process.env.CI_CEPH !== undefined;
```

Comment:
can this be simply, `const isCEPH = process.env.CI_CEPH` ?

Reply:
it's the same, except your method is not making the variable a boolean :)